CMakeLists.txt

```diff
@@ -0,0 +1,455 @@
+#
+# CMakeLists.txt
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.5)
+project(fdb)
```

Comment:
I see that we are getting version information from `versions.target`. As we are moving to a new world, can we use version argument of `project()` instead?

Reply:
I would love to. The reason I didn't do this yet, is because I want this to work for both build systems (for now)